TerriaJS/terriajs
# Misc fixes

### Misc fixes

#### Fix `TileErrorHandlerMixin.tileRetryOptions` type

Just type issue - fixes error when building specs

#### Fix `BottomDock.handleClick` binding

We were getting these errors

<img width="1221" alt="image" src="https://user-images.githubusercontent.com/6187649/172036122-3d018979-313d-4e4d-99e5-4f1c07ae8dac.png">

To test:
- Map settings
- "Always show timeline"
- Click on timeline
- Observe no errors 

### Checklist
